programs/lighthouse/src/utils.rs

```diff
@@ -68,6 +89,22 @@ pub fn try_from_slice<T: BorshDeserialize + Sized>(
     Ok(T::try_from_slice(slice)?)
 }
 
+pub fn try_from_slice_pubkey(data: &[u8], offset: usize) -> Result<&Pubkey> {
```

Comment:
Why do you need this? Why not just use `try_from_slice`? 

Alternatively, can you have this be a simple wrapper over `try_from_slice`?

Reply:
The only difference is it uses bytemuck so it doesnt have to copy but I could make it more generic. The main thing with all these utility functions is they have safety checks and log messages so make the rest of the code cleaner.